configs/components/marionette-collective.json

```diff
@@ -1 +1 @@
-{"url": "git://github.com/puppetlabs/marionette-collective.git", "ref": "a28c1afe18b619c05e18c98b68007e4327fd425c"}
+{"url": "git://github.com/puppetlabs/marionette-collective.git", "ref": "refs/tags/2.8.7"}
```

Comment:
It looks like puppet-agent 1.3.2 shipped with mco 2.8.6, but based on https://confluence.puppetlabs.com/display/PROD/puppet-agent+Versions+and+Dependencies#puppet-agentVersionsandDependencies-puppet-agent1.3.z it looks like we do want mco 2.8.7, and there is a 2.8.7 tag, so :+1: